ConfigSpace/util.pyx

```diff
@@ -436,3 +436,185 @@ def fix_types(configuration: dict,
             else:
                 raise TypeError("Unknown hyperparameter type %s" % type(param))
     return configuration
+
+cimport cython
+@cython.boundscheck(True)  # Activate bounds checking

```

Comment:
Great that you figured out how to only change the settings for only this function!

Reply:
Yes, the link here: [https://cython.readthedocs.io/en/latest/src/userguide/numpy_tutorial.html#tuning-indexing-further](https://cython.readthedocs.io/en/latest/src/userguide/numpy_tutorial.html#tuning-indexing-further) seems to suggest array lookups _are_ slowed down by deactivating those.